### PR TITLE
Add Python reference implementation

### PR DESCRIPTION
**Pull request by dstebila**

### CODE REVIEW

File: python3/frodokem.py [hunk 1]
```diff
@@ -0,0 +1,615 @@
...
+        sk.append(pkh)
+        sk = sk.bytes
+        assert len(sk) == self.len_sk_bytes
+        return (pk, sk)
+
+    def kem_encaps(self, pk):
+        """Encapsulate against a public key to create a ciphertext and shared secret 
+        (FrodoKEM specification, Algorithm 13)"""
+        # Parse pk = seedA || b
+        assert len(pk) + self.len_seedA_bytes + self.D * self.n * self.nbar, "Incorrect public key length"
```
Comment (ilyamironov) [inline python3/frodokem.py]:
```suggestion
        assert len(pk) == self.len_seedA_bytes + self.D * self.n * self.nbar / 8, "Incorrect public key length"
```

File: README.md [hunk 2]
```diff
@@ -19,19 +19,19 @@ The library was developed by [Microsoft Research](http://research.microsoft.com/
```
[the remark this answers is not in the document]
Comment (dstebila) [reply README.md]: I'll leave that to @patricklonga to respond to.

File: python3/frodokem.py [hunk 3]
```diff
@@ -0,0 +1,615 @@
+# Creative Commons Zero v1.0 Universal
+# SPDX-License-Identifier: CC0-1.0
+# Created by Douglas Stebila
+
+import bitstring
+import math
```
[the remark this answers is not in the document]
Comment (dstebila) [reply python3/frodokem.py]: Fixed in #14

File: python3/frodokem.py (hunk 1 above)
Comment (dstebila) [reply python3/frodokem.py]: Fixed in #14

**Pull request merged**